src/content/docs/en/pages/devtools/cli/azion-cli-alpha/cache-settings/cache-settings.mdx

```diff
@@ -1,12 +1,16 @@
 ---
-title: Cache Settings
+title: Azion CLI cache-setting
 description: Create, list, describe, update, and delete your cache settings on Azion's platform through Azion CLI.
 meta_tags: cli, cache, devops
 namespace: documentation_cli_cache_settings
 menu_namespace: cliMenuAlpha
 permalink: /documentation/products/cli/cache-settings-crud/
 ---
 
+:::note 
+In case one or more required fields aren't informed through the specific flags, an interactive message prompts and asks for the missing information.
```

Comment:
```suggestion
In case any required field isn't informed through the specific flags, an interactive message prompts and asks for the missing information.
```

Reply:
tentei deixar mais genérico e direito... entendo que desde o primeiro campo não informado, aparece a msg...